web-ng/src/app/services/navigation/navigation.service.ts

```diff
@@ -38,6 +38,7 @@ export class NavigationService {
   static readonly PROJECT_ID = 'projectId';
   static readonly PROJECT_SEGMENT = 'project';
   static readonly SIGN_IN_SEGMENT = 'signin';
+  static readonly HOME_SEGMENT = 'home';
```

Comment:
`PROJECTS_SEGMENT = 'projects'`

would be a more idiomatic way of representing the projects list. (Thinking of the URL as a path the a resource like a single project or list of projects rather than a page)

Reply:
updated.